docs/content/concepts/authorization.md

```diff
@@ -208,9 +208,59 @@ and work just like in a regular Kubernetes cluster.
 
 It is possible to bind to roles and cluster roles in the bootstrap policy from a local policy `RoleBinding` or `ClusterRoleBinding`.
 
+### Scopes
+
+Scopes are a way to limit the access of a user to a specific logical cluster. Scopes are attached to the user identity
+by adding `cluster:<logical-cluster>` to the `authentication.kcp.io/scope` extra field. The scope is then checked by
+the authorizers. For example:
+
+```yaml
+user: user1
+groups: ["group1"]
+extra:
+  authentication.kcp.io/scope: cluster:logical-cluster-1
+```
+
+This user will only be allowed to access resources in `logical-cluster-1`.
+
 ### Service Accounts
 
 Kubernetes service accounts are granted access to the workspaces they are defined in and that are ready.
 
 E.g. a service account "default" in `root:org:ws:ws` is granted access to `root:org:ws:ws`, and through the
 workspace content authorizer it gains the `system:kcp:clusterworkspace:access` group membership.
+
+Service accounts from foreign workspaces are authenticated as user `system:kcp:serviceaccount:<logicalcluster>:<ns>:<serviceaccount>` 
+and under that name can be granted access to workspaces and other permissions. The original service account name is 
+appended as a warrant (see next section) in order to match the same (cluster) role bindings.
+
+### Warrants
+
+Warrants are a way to grant extra access to a user. It can be limited by the scope of a logical cluster.
+A warrant is attached by adding a `authorization.kcp.io/warrant` extra field to the user identity
+with a JSON-encoded user info, and the limiting logical cluster set as `authentication.kcp.io/scope: cluster:<logical-cluster>`.
+in the embedded user info's extra. The warrant is then checked by the authorizers in the chain of every step
+if the primary users is not allowed. For example:
+
+```yaml
+user: user1
+groups: ["group1"]
+extra:
+  authorization.kcp.io/warrant: |
```

Comment:
` authorization.kcp.io/warrants` as It can the list?

Reply:
fixed